src/qml/controls/IPAddressValueInput.qml

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2024 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+TextInput {
+    id: root
+    required property string parentState
+    property string description: ""
+    property bool filled: false
+    property int descriptionSize: 18
+    property color textColor: root.filled ? Theme.color.neutral9 : Theme.color.neutral5
+    // Expose a property to indicate validity, initial value will be true (no error message displayed)
+    // if both properties errortText and description are set
+    property bool validInput: description!=="" && errorText!==""
```

Comment:
errorText isn't within this scope. I think just initializing to true works here.
```suggestion
    property bool validInput: true
```

Reply:
Ok.